app/helpers/application_helper.rb

```diff
@@ -252,4 +252,20 @@ def allowed_boards(obj, user)
     authored_ids = BoardAuthor.where(user: user).select(:board_id)
     Board.where(id: obj.board_id).or(Board.where(authors_locked: false)).or(Board.where(id: authored_ids)).ordered
   end
+  def get_owed
+    ids = Post::Author.where(user_id: current_user.id, can_owe: true).group(:post_id).pluck(:post_id)
+    @posts = Post.where(id: ids)
+    drafts = ReplyDraft.where(post_id: @posts.select(:id)).where(user: current_user).pluck(:post_id)
```

Comment:
You should be able to use `ids` here instead of `@posts.select(:id)` to make it more efficient